app/javascript/widget/assets/scss/_utilities.scss

```diff
@@ -0,0 +1,3 @@
+.icon-button {
+  @include button-size;
```

Comment:
Since tailwind is already there, we could stick to `w-8` and `h-8` for the same size.